.travis.yml

```diff
@@ -26,6 +26,15 @@ env:
       AFTER_SCRIPT="./.coverage.sh"
       CMAKE_ARGS="-DENABLE_COVERAGE_TESTING=ON -DCMAKE_BUILD_TYPE=Debug"
       DOCKER_RUN_OPTS='-e COVERAGE_PKGS="pilz_control pilz_utils prbt_hardware_support"'
+    - ROS_REPO="testing"
+      CATKIN_LINT=false ADDITIONAL_DEBS="clang" CXX=clang++ CC=clang CMAKE_ARGS="-DCMAKE_BUILD_TYPE=Debug"
```

Comment:
```suggestion
      CATKIN_LINT=false ADDITIONAL_DEBS="clang" CXX=clang++ CC=clang CMAKE_ARGS="-DCMAKE_BUILD_TYPE=Debug -DENABLE_ADDRESS_SANITIZING"
```

Reply:
Something like this could simplify the CMAKE checks

Reply:
I agree. But what should I do now? Let's make these kind of improvements later on. One step after another ;) -> Done.

Reply:
Agree if you open a reminder issue

Reply:
See #282.